### PR TITLE
SMP-1350: changes for timescaledb functions

### PR DESCRIPTION
**Pull request by arya-harness**

### CODE REVIEW

File: src/common/templates/_databaseconnections.tpl [hunk 1]
```diff
@@ -120,7 +120,20 @@
...
+{{- if or .args $extraArgs }}
+{{- $appendedArgs = (printf "%s?" $appendedArgs)}}
+{{- end }}
+{{- if .args }}
+{{- $appendedArgs = (printf "%s%s" $appendedArgs .args )}}
+{{- end }}
+{{- if and .args $extraArgs }}
+{{- $appendedArgs = (printf "%s&" $appendedArgs)}}
+{{- end }}
+{{- $appendedArgs = (printf "%s%s" $appendedArgs $extraArgs )}}
```
Comment (kapilgarg1996) [inline src/common/templates/_databaseconnections.tpl:134]:
```suggestion
{{- $finalArgs := (printf "/%s" .database) }}
{{- if and .args $extraArgs }}
{{- $finalArgs = (printf "%s?%s&%s" $finalArgs .args $extraArgs) }}
{{- else }}
{{- $finalArgs = (printf "%s?%s" $finalArgs (default .args $extraArgs)) }}
{{- end }}
```

Comment (arya-harness) [reply src/common/templates/_databaseconnections.tpl:134]: will not work if both the args are missing

Comment (kapilgarg1996) [reply src/common/templates/_databaseconnections.tpl:134]: just tweak it a bit, the goal is to simplify the logic

Comment (kapilgarg1996) [reply src/common/templates/_databaseconnections.tpl:134]: I tweaked it

Comment (arya-harness) [reply src/common/templates/_databaseconnections.tpl]: Done

**Review by kapilgarg1996 — APPROVED** [review]

**Pull request merged**